### PR TITLE
add 4.8 Operator manifests

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: manifests/4.8/ptp-operator.v4.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,258 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (fedepaol) [inline manifests/4.8/ptp-operator.v4.7.0.clusterserviceversion.yaml:1]: Wrong file name?

**Comment by zshi-redhat** [comment]
LGTM other than the issue found by Federico

/cc @huiran0826

File: manifests/4.8/ptp-operator.v4.7.0.clusterserviceversion.yaml (hunk 1 above)
Comment (SchSeba) [reply manifests/4.8/ptp-operator.v4.7.0.clusterserviceversion.yaml]: sorry about that

**Comment by SchSeba** [comment]
@fedepaol @zshi-redhat can you have another look please?

**Pull request merged**